proposed/cache.md

```diff
@@ -293,11 +293,13 @@ interface CacheItemPoolInterface
      *
      * @param string $key
      *   The key for which to return the corresponding Cache Item.
-     * @return \Psr\Cache\CacheItemInterface
-     *   The corresponding Cache Item.
-     * @throws \Psr\Cache\InvalidArgumentException
+     *
+     * @throws InvalidArgumentException
      *   If the $key string is not a legal value a \Psr\Cache\InvalidArgumentException
      *   MUST be thrown.
+     *
+     * @return CacheItemInterface
+     *   The corresponding Cache Item.
```

Comment:
If PSR-5 is likely to define it this way then lets just put it this way. This is trivial from a standards perspective while still being consistent. Plus I have to agree with Graham, I always see return as the last position.